### PR TITLE
Drop support for run/debug of deprecated runtimes

### PR DESCRIPTION
**Pull request by abrooksv**
Sam has removed support as of 0.49

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by abrooksv** [comment]
Updated error handling to provide better error:
Screen Shot 2020-05-18 at 2 40 38 PM

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt [hunk 1]
```diff
@@ -289,16 +289,21 @@ class LocalLambdaRunConfiguration(project: Project, factory: ConfigurationFactor
...
-        runtime ?: throw RuntimeConfigurationError(message("lambda.run_configuration.no_runtime_specified"))
-        return Triple(handler, runtime, null)
+
+        return Triple(handler, runtime.validateSupportedRuntime(), null)
+    }
+
+    private fun Runtime?.validateSupportedRuntime(): Runtime = if (this?.runtimeGroup != null) {
+        this
+    } else {
+        throw RuntimeConfigurationError(message("lambda.run_configuration.no_runtime_specified"))
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt:306]: can we go one step further and say "node8.10" is not a supported runtime?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt [hunk 2]
```diff
@@ -289,16 +289,21 @@ class LocalLambdaRunConfiguration(project: Project, factory: ConfigurationFactor
         val handler = tryOrNull { function.handler() }
             ?: throw RuntimeConfigurationError(message("lambda.run_configuration.no_handler_specified"))
 
-        val runtime = tryOrNull { Runtime.fromValue(function.runtime()).validOrNull }
-            ?: throw RuntimeConfigurationError(message("lambda.run_configuration.no_runtime_specified"))
+        val runtime = Runtime.fromValue(function.runtime()).validOrNull.validateSupportedRuntime()
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt:292]: similar thing here and in `resolveLambdaInfo`, would be nice to tell the user if they entered a runtime that isn't modeled in the SDK

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt:292]: Make sense, will improve them and post new revision

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt (hunk 1 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt]: Done

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt (hunk 2 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt]: We pass through their value now

**Pull request merged**